pkgs/development/python-modules/rospkg/setup-hook.sh

```diff
@@ -0,0 +1 @@
+export ROS_OS_OVERRIDE=nixos
```

Comment:
this almost certainly should be in `env` instead

Reply:
Thanks for trying. I have never had to use it, that's why I was initially surprised. All good!